.circleci/config.yml

```diff
@@ -3,17 +3,17 @@ jobs:
   build:
     working_directory: /go/src/github.com/Financial-Times/draft-annotations-api
     docker:
-      - image: golang:1.12
+      - image: golang:1.12.9
```

Comment:
This is a degradation from the previous version of the line because "golang:1.12" resolves to the latest 1.12 version. E.g. if the latest version is 1.12.10 it will get that while 1.12.9 will not.

Furthermore lets use "golang:1" according to our latest guidelines.